### PR TITLE
Tickets/preops 582

### PR DESCRIPTION
**Pull request by jeffcarlin**
For some reason the results set from merging two queries as pandas dataframes has one more entry than the search using a JOIN in the ADQL. I'm not sure why that is - suggestions are welcome. Otherwise, we could just add a statement about this in the notebook?

### CODE REVIEW

File: 06_Comparing_Object_and_Truth_Tables.ipynb [hunk 1]
```diff
@@ -159,124 +161,88 @@
...
+    "\n",
+    "Note that for the Object table we select all objects within the cone-shaped region of interest. In the Truth-Match table, we restrict the results to objects satisfying \"match_objectId >= 0 AND is_good_match = 1\". According to the Truth-Match schema above, the `is_good_match` flag is \"True if this object--truth matching pair satisfies all matching criteria\" as laid out in the [DESC DC2 Release Note](https://ui.adsabs.harvard.edu/abs/2021arXiv210104855L/abstract). We'll use that to select \"good\" matches. In the column description for `match_objectId` from above, it says \"objectId of the matching object entry (-1 for unmatched truth entries).\" Thus the criterion \"match_objectId >= 0\" removes the unmatched entries, leaving us with only the truth-table entries that were detected and appear in the Object table.\n"
```
Comment (leannep) [inline 06_Comparing_Object_and_Truth_Tables.ipynb:173]: Check that this (and other above) line lengths do not violate the coding guidelines.

File: 06_Comparing_Object_and_Truth_Tables.ipynb [hunk 2]
```diff
@@ -92,7 +94,7 @@
   },
```
Comment (leannep) [inline 06_Comparing_Object_and_Truth_Tables.ipynb:94]: Remove commented code

**Review by leannep — CHANGES_REQUESTED** [review]

File: 06_Comparing_Object_and_Truth_Tables.ipynb (hunk 1 above)
Comment (jeffcarlin) [reply 06_Comparing_Object_and_Truth_Tables.ipynb:173]: Markdown cells aren't checked by the notebook versions of flake8. The only lines in this notebook that exceed the line length guidelines are in the last two plotting cells. But for readability, I think they are best left as they are.

File: 06_Comparing_Object_and_Truth_Tables.ipynb (hunk 2 above)
Comment (jeffcarlin) [reply 06_Comparing_Object_and_Truth_Tables.ipynb:92]: Thanks for catching that. I removed a couple instances of commented code throughout.

**Review by leannep — APPROVED** [review]

**Pull request merged**